src/Core/src/Platform/Windows/KeyboardAcceleratorExtensions.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.System;
+
+namespace Microsoft.Maui.Platform
+{
+	public static class KeyboardAcceleratorExtensions
+	{
+		public static void UpdateAccelerator(this MenuFlyoutItemBase platformView, IMenuFlyoutItem menuFlyoutItem)
+		{
+			var keyboardAccelerators = menuFlyoutItem.Accelerator.ToPlatform();
+
+			if (keyboardAccelerators is null)
+				return;
+
+			foreach (var keyboardAccelerator in keyboardAccelerators)
+				platformView.KeyboardAccelerators.Add(keyboardAccelerator);
+			
+		}
+
+		// Single key (A, Delete, F2, Spacebar, Esc, Multimedia Key) accelerators and multi-key
+		// accelerators (Ctrl+Shift+M) are supported.
+		// Gamepad virtual keys are not supported.
+		public static IList<KeyboardAccelerator>? ToPlatform(this IAccelerator accelerator)
+		{
+			if (accelerator is null)
+				return null;
+
+			List<KeyboardAccelerator> result = new List<KeyboardAccelerator>();
+			
+			var keys = accelerator.Keys;
+
+			if (keys is not null)
+			{
+				var keysCount = keys.Count();
+				bool hasKeys = keysCount > 0;
+
+				if (hasKeys)
+				{
+					var key = keys!.ElementAt(0);
+					var modifiers = accelerator.Modifiers;
+
+					if (modifiers is not null)
+					{
+						var modifiersCount = modifiers.Count();
+						bool hasModifierMask = modifiersCount > 0;
+
+						if (hasModifierMask)
+						{
+							if (modifiersCount == 1)
+								result.Add(CreateSingleKeyAccelerator(modifiers!.ElementAt(0), key));
+							else
+								result.Add(CreateMultiKeyAccelerator(modifiers!, key));
+						}
+					}
+					else
+						result.Add(CreateSingleKeyAccelerator(string.Empty, key));
+
+				}
+			}
+
+			return result;
+		}
+
+		internal static VirtualKeyModifiers ToVirtualKeyModifiers(this string modifierMask)
+		{
+			switch (modifierMask.ToLowerInvariant())
+			{
+				case "ctrl":
+					return VirtualKeyModifiers.Control;
+				case "alt":
+					return VirtualKeyModifiers.Menu;
+				case "shift":
+					return VirtualKeyModifiers.Shift;
+				case "win":
```

Comment:
What about macos? Do we need a cmd and opt?

Reply:
Yes, is coming, working in the macOS impl with @rachelkang